investor/LoanFilter.py

```diff
@@ -23,16 +24,11 @@ def __str__(self):
 		pass
 
 	@abstractmethod
-	def _eval(self, loan):
+	def _eval(self, loan, block):
 		pass
 
-	def apply(self, loan):
-		if self._eval(loan):
-			self.pass_count += 1
-			return True
-		else:
-			self.fail_count += 1
-			return False
+	def apply(self, loan, block=True):
+		return self._eval(loan, block)
```

Comment:
We lose pass and fail counts by doing this. I'm not sure when we'd use them at this point, but I'd like to keep the functionality alive as long as we're keeping the pass/fail_count instance variables.